community/index/src/test/java/org/neo4j/index/gbptree/GBPTreeIT.java

```diff
@@ -59,17 +54,20 @@
 import static org.junit.Assert.fail;
 import static org.neo4j.index.IndexWriter.Options.DEFAULTS;
 import static org.neo4j.index.gbptree.GBPTree.NO_MONITOR;
-import static org.neo4j.io.pagecache.tracing.PageCacheTracer.NULL;
+import static org.neo4j.index.gbptree.PageCacheTestUtil.pageCache;
```

Comment:
The PageCacheRule lives in kernel and is somewhat tied to other things in the kernel so we opted for not moving it. Things from it moved to neo4j-io tho, so we're better off than before. The page cache management in these tests I think is minimal and manageable.